op-service/txmgr/cli.go

```diff
@@ -79,7 +79,6 @@ var (
 		MinBaseFeeGwei:            1.0,
 		ResubmissionTimeout:       48 * time.Second,
 		NetworkTimeout:            10 * time.Second,
-		TxSendTimeout:             10 * time.Minute,
```

Comment:
How about we explicitly set it to `nil` and add a comment explaining that it is due to Holocene rules?